src/CssToInlineStyles.php

```diff
@@ -110,27 +129,61 @@ public function getInlineStyles(\DOMElement $element)
      * @return \DOMDocument
      */
     protected function createDomDocumentFromHtml($html)
+    {
+        return null !== $this->html5Parser ? $this->parseHtml5($html) : $this->parseXhtml($html);
+    }
+
+    /**
+     * @param string $html
+     * @return \DOMDocument
+     */
+    protected function parseHtml5($html)
+    {
+        $this->isHtml5Document = strspn($html, " \t\r\n") === stripos($html, '<!doctype html>');
```

Comment:
this looks wrong to me. It is still using the HTML5 parser even when this is `false`

Reply:
I think the way I implemented is the same as `symfony/dom-crawler`. It did seem a little odd to me. I'll change it if you want?

Reply:
I was looking at an old version. I've updated it :+1: